lib/opentok.js

```diff
@@ -515,13 +569,72 @@ OpenTok = function (apiKey, apiSecret, env) {
     }
     else {
       options.sessionId = sessionId;
+      if (!options.streamMode) {
+        options.streamMode = 'auto';
+      }
       client.startBroadcast(options, function (err, json) {
         if (err) return callback(new Error('Failed to start broadcast. ' + err));
         return callback(null, new Broadcast(client, json));
       });
     }
   };
 
+  /**
+  * Updates a {@link Broadcast} object with either addStream or removeStream
+  *
+  * @param broadcastId {String} The archive ID.
+  *
+  * @param options {Object} An options parameter with either of these two properties properties:
+  *
+  * <ul>
+  *
+  *   <li>
+  *     <code>addStream</code> &mdash; The stream to add to archiving
+  *   </li>
+  *
+  *   <li>
+  *     <code>removeStream</code> &mdash; Teh stream to remove from archiving
+  *   </li>
+  *
+  * </ul>
+  *
+  * If you pass the <code>addStream</code> argument, there are two other optional arguments:
+  *
+  * <ul>
+  *
+  *   <li>
+  *     <code>hasAudio</code> &mdash; Archive the audio stream. Default value is true.
+  *   </li>
+  *
+  *   <li>
+  *     <code>hasAudio</code> &mdash; Archive the video stream. Default value is true.
+  *   </li>
+  *
+  * </ul>
+  * @method #patchBroadcast
+  * @memberof OpenTok
+  */
+  this.patchBroadcast = function patchBroadcast(broadcastId, options, callback) {
```

Comment:
This does not match the proposal: https://github.com/opentok/api-review/pull/232/. The proposal has two methods: `OpenTok.addBroadcastStream()` and `OpenTok.removeBroadcastStream()`.